### PR TITLE
browser: final fixes 

### PR DESCRIPTION
**Pull request by skylarbarrera**
- improves dapp query to allow selectors
- switched the more button on the control panel to favorites 
- added the fetching of onchain native balances for the tx sheet to reduce stale data
- hide and resizes control panel when on the hompage

### CODE REVIEW

File: src/components/DappBrowser/control-panel/ControlPanel.tsx [hunk 1]
```diff
@@ -610,11 +615,7 @@ const ListHeader = React.memo(function ListHeader({
   return (
     <Box style={controlPanelStyles.listHeader}>
       <Box style={controlPanelStyles.listHeaderContent}>
-        <ButtonPressAnimation
-          onPress={goBack}
-          scaleTo={0.8}
-          style={[controlPanelStyles.listHeaderButtonWrapper, { backgroundColor: 'red' }]}
-        >
+        <ButtonPressAnimation onPress={goBack} scaleTo={0.8} style={[controlPanelStyles.listHeaderButtonWrapper]}>
```
Comment (brunobar79) [inline src/components/DappBrowser/control-panel/ControlPanel.tsx:618]: Idk why we had a red bg color here

File: src/screens/SignTransactionSheet.tsx [hunk 2]
```diff
@@ -354,8 +355,14 @@ export const SignTransactionSheet = () => {
   useEffect(() => {
     (async () => {
       const asset = await ethereumUtils.getNativeAssetForNetwork(currentNetwork, accountInfo.address);
-      if (asset) {
-        provider && setNativeAsset(asset);
+      if (asset && provider) {
+        const balance = await getOnchainAssetBalance(asset, accountInfo.address, currentNetwork, provider);
+        if (balance) {
+          const assetWithOnchainBalance: ParsedAddressAsset = { ...asset, balance };
+          provider && setNativeAsset(assetWithOnchainBalance);
```
Comment (brunobar79) [inline src/screens/SignTransactionSheet.tsx:362]:
```suggestion          
          setNativeAsset(assetWithOnchainBalance);
```

File: src/screens/SignTransactionSheet.tsx [hunk 3]
```diff
@@ -354,8 +355,14 @@ export const SignTransactionSheet = () => {
...
+        } else {
+          provider && setNativeAsset(asset);
```
Comment (brunobar79) [inline src/screens/SignTransactionSheet.tsx:364]:
```suggestion
          setNativeAsset(asset);
```

File: src/components/DappBrowser/control-panel/ControlPanel.tsx (hunk 1 above)
Comment (skylarbarrera) [reply src/components/DappBrowser/control-panel/ControlPanel.tsx:618]: ;)

**Pull request merged**